packages/yosqlite/yosqlite.0.2/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Simple Sqlite3. Yojson+Sqlite3 amalgamation"
+description: "Convenience functions for working with Sqlite3 library"
+maintainer: ["Szymon Wygnański"]
```

Comment:
Would it be possible to add a contact email or you prefer people to open directly issues on github?